study.md

```diff
@@ -23,13 +23,13 @@ Array methods?
 Similarities:
 
 ```md
-<!-- your answer here -->
+For both Ruby and JS, the push method is used to add values to the end of an existing array.  In addition, they both modify an existing array rather than create a new one.
 ```
 
 Differences:
 
 ```md
-<!-- your answer here -->
+The only difference I was able to determine is the syntax.  While JS requires a variable be declared (with let) before being applied (and ending with a semicolon), they code is virtually the same.
```

Comment:
Ruby's push array method returns the modified array, allowing method chaining.

JavaScript's push array method returns the length of modified array.